### DISCUSSION TITLE
[Script request] NetAlertX - new name for PiAlert

### DISCUSSION THREAD

**Discussion by ricostuart** [community-scripts/ProxmoxVE#188] [Request script] [answered]
### Application Name

NetAlertX

### Website

https://github.com/jokob-sk/NetAlertX

### Description

Pi.Alert has been archived now and upgraded to the new NetAlertX on github. If either the install is changed to the new version, or a replacement made instead, that would be great

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Mithias-uk** [comment]
I'd posted about this on the old discussion board but haven't moved it over so thank you. Would be great to get this going. Only oddity is that the "HW install" as they call it is experimental and the official supported way is Docker - which I've tried but would MUCH prefer not to nest where possible. 

The changes in this from the original PiAlert look great though

**Comment by MickLesk** [comment]
We use this repo:
https://github.com/leiweibau/Pi.Alert/

That doesn't look like it's archived ^^

**Reply by ricostuart** [reply]
Sorry, yeah the main repo isnt, but the brached repo is. My mistake there. But still think NetAlertX might be worth looking into.

**Comment by Mithias-uk** [comment]
Yeah its not archived, NetAlertX is a fork (if you can call it that) with some enhanced functionality

________________________________

**Comment by Zazarothh** [comment]
Friendly bump <3 thank y'all for what you do

**Comment by tremor021** [comment, marked as answer]
The bare metal install for this app is highly experimental, prone to breaking and not supported by the developer. Even though he added some instructions how to make it run.

As much as  i think we would like to have this as a LXC, i'm afraid that it would just lead to many reports of broken containers or lost data.
Maybe once developer makes a more stable version that can reliably work on bare metal, sure we can take a look and make it a LXC.

**Reply by Zazarothh** [reply]
Solid update. Thank you tons

**Reply by tremor021** [reply]
In the meantime LeiWeibau's fork that we have as LXC works very good. I can't remember we ever had a issue with that app :)

**Comment by v3rm1n0** [comment]
NetAlertX seems to have an install script for proxmox made by them selfs.

Also its not "unsupported" it just relies on community support, as far as I understand it they just find ich controversial to pipe to bash [...]
> as you cannot see the code that's about to be executed on your system. 
[...]

Anyways you can find the script here:

https://github.com/jokob-sk/NetAlertX/blob/main/docs/HW_INSTALL.md#-bare-metal---proxmox

**Reply by tremor021** [reply]
Yea, that doesnt work.

`git clone -b baremetal-installer https://github.com/jokob-sk/NetAlertX.git "$INSTALL_DIR/"`

this branch doesn't exist

**Reply by v3rm1n0** [reply]
Yeah just realised that too, I wrote them a message on discord regarding that

**Comment by oerix** [comment]
Just wondering, if there is any update on this please? Would be a great addition!

**Reply by MickLesk** [reply]
For what? We have it

**Reply by oerix** [reply]
> For what? We have it

Where do you have it? These are two different forks, with NetAlertX being more feature rich.

Pi.Alert: https://github.com/leiweibau/Pi.Alert

NetAlertX: https://github.com/jokob-sk/NetAlertX?

**Reply by MickLesk** [reply]
If we were to take every fork to an script, we would have 800 scripts. If anyone else wants to do it, that's fine with me, but we won't be doing it because we already have an active script for it.
